v3-nginx/Dockerfile-alpine

```diff
@@ -157,6 +156,8 @@ RUN set -eux; \
         lmdb-dev \
         moreutils \
         tzdata \
+        pcre \
+        pcre2 \
```

Comment:
Why do we need to install those here but not on Debian?

Reply:
I can try to see ldd in debian. But the TL;DR here is that alpine is failing to start, but debian is working 🤷

Reply:
No, it's fine. I just wanted to make sure that it wasn't an oversight.